dataland-frontend/src/components/resources/dataTable/conversion/YesNoDataPointValueGetterFactory.ts

```diff
@@ -0,0 +1,43 @@
+import { type Field } from "@/utils/GenericFrameworkTypes";
+import { type AvailableMLDTDisplayObjectTypes } from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import { type YesNoNa } from "@clients/backend";
+import { getDataPointGetterFactory } from "@/components/resources/dataTable/conversion/Utils";
+import { type GenericDataPoint } from "@/utils/DataPoint";
+
+const humanReadableYesNoMap: { [key in YesNoNa]: string } = {
+  Yes: "Yes",
+  No: "No",
+  NA: "N/A",
+};
+
+const certificateHumanReadableYesNoMap: { [key in YesNoNa]: string } = {
+  Yes: "Certified",
+  No: "Uncertified",
+  NA: "N/A",
+};
+
+/**
+ * Returns a value factory that returns the value of the Yes / No Data Point form field
+ * @param path the path to the field
+ * @param field the field
+ * @returns the created getter
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Again not clear what any is being suppressed.

Reply:
Absolutely. Removed together with 2 more such occurrences.